spec/application_controller_spec.rb

```diff
@@ -11,6 +11,52 @@ def app
     expect(last_response.status).to eq(200)
     body = JSON.parse(last_response.body, symbolize_names: true)
 
-    expect(body[:message]).to include("Welcome to the Sinatra Template!")
+    expect(body[:message]).to include("Welcome to the Yelp Microservice!")
+  end
+
+  xit 'can get utility data' do
```

Comment:
We will have to look back into why this test fails on Travis but not on local after deploying to Heroku!